cromshell

```diff
@@ -384,20 +438,27 @@ if ${ISCALLEDBYUSER} ; then
     exit $?
   fi
 
+	# Special case: list
+	if [[ "${SUB_COMMAND}" == "list" ]] ; then
+		${SUB_COMMAND}
+		exit $?
+	fi
+	
   # Special case: notify
   if [[ "${SUB_COMMAND}" == "notify" ]] ; then
 
     # Is the next argument our workflow ID?
     # Workflow IDs are standard UUIDs:
     if [[ "${1}" =~ ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$ ]] ; then
       # Get the workflow ID:
-      populateWorkflowId $1
+      populateWorkflowIdAndServerUrl $1
       shift
     fi
  
     # Get the host on which this will be run:
     hostServer="${HOSTNAME}"
-    if [[ $# -gt 1 ]] ; then
+    
+		if [[ ! "${1}" =~ ^.*@.*  ]] ; then
```

Comment:
is this matching something with an `@` in it?

Reply:
Yes.  The full purpose (with the negation) is to detect if the next argument is NOT the email address to which to send the notification.